src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/skjema/SkjemaInntektsmelding.kt

```diff
@@ -38,47 +60,59 @@ data class SkjemaInntektsmelding(
 
             avsender.valider(),
             agp?.valider(),
-            inntekt?.valider(),
+            inntekt.valider(),
             refusjon?.valider(),
 
-            // Må sjekke sykmeldingsperioder fordi beregning av bestemmende fraværsdag krever ikke-tom liste
-            if (agp != null && inntekt != null && sykmeldingsperioder.isNotEmpty()) {
-                val bestemmendeFravaersdag = bestemmendeFravaersdag(
-                    arbeidsgiverperioder = agp.perioder,
-                    sykmeldingsperioder = sykmeldingsperioder,
-                )
+            validerBestemmendeFravaersdagMotInntektsdato(agp, inntekt, sykmeldingsperioder),
+            validerRefusjonMotInntekt(refusjon, inntekt),
+        )
+            .tilFeilmeldinger()
+}
 
-                val feiletValidering = valider(
-                    vilkaar = !bestemmendeFravaersdag.isBefore(inntekt.inntektsdato),
-                    feilmelding = Feilmelding.TEKNISK_FEIL,
-                )
+private fun validerBestemmendeFravaersdagMotInntektsdato(
```

Comment:
Denne kan vi ikke bruke på skjema for forespurt IM fordi den mangler sykmeldingsperioder. Vi kan heller sjekke det senere i løypa.